lib/waterline/model/lib/defaultMethods/save.js

```diff
@@ -29,6 +33,7 @@ module.exports = function(context, proto, cb) {
     deferred = defer();
   }
 
+  options = options || optDefaults;
```

Comment:
Good call! I updated the code to use this. Admittedly I have never used lodash, so thanks for the heads up!